### PR TITLE
feat: user story 40

### PR DESCRIPTION
**Pull request by ross-ian28**
## Description of Changes
Added new test for user story 40, passes.
## What gif describes how you feel about this PR?
ferret_dance

### CODE REVIEW

File: app/controllers/merchant_dashboard_controller.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
-class MerchantsController < ApplicationController
+class  MerchantDashboardController < ApplicationController
   def index
-
+    #binding.pry
```
Comment (jgrazulis) [inline app/controllers/merchant_dashboard_controller.rb]: I spy a pry

File: spec/features/merchants/dashboard_spec.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe 'User story 40' do
+  before(:each) do
+    @merchant1 = Merchant.create!(name: "Pabu")
+    visit "/merchants/#{@merchant1.id}/dashboard"
+  end
+  it 'shows every merchant' do
+    save_and_open_page
```
Comment (jgrazulis) [inline spec/features/merchants/dashboard_spec.rb]: ooh and a save_and_open_page

**Review by jgrazulis — CHANGES_REQUESTED** [review]
Looks great, one user story down!

**Review by jgrazulis — APPROVED** [review]
Awesome, thanks Ian!

**Review by glsorensen — APPROVED** [review]

**Pull request merged**